sdk/js/src/Client.ts

```diff
@@ -40,10 +40,14 @@ export class DVCClient implements Client {
         this.requestConsolidator = new RequestConsolidator()
         this.eventEmitter = new EventEmitter()
 
+        if (options?.enableCloudData) {
```

Comment:
this also needs to happen if the project config includes the `enableCloudEntityData` setting